tests/performance/utils.go

```diff
@@ -0,0 +1,91 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ *
+ */
+
+package performance
+
+import (
+	"fmt"
+	"time"
+
+	"math"
+	"math/rand"
+
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	kvv1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/tests"
+	cd "kubevirt.io/kubevirt/tests/containerdisk"
+	e2emetrics "kubevirt.io/kubevirt/tests/performance/metrics"
+)
+
+var r *rand.Rand
+
+// poissonProcess returns arrival interval time for a given lambda
+func poissonProcess(lambda float64) time.Duration {
+	if r == nil {
+		// init random number generator
+		r = rand.New(rand.NewSource(99))
+	}
+
+	randNumber := r.Float64()
+	inter_arrival_time := -math.Log(1.0-randNumber) / float64(lambda)
```

Comment:
```suggestion
	interArrivalTime := -math.Log(1.0-randNumber) / float64(lambda)
```

Reply:
@rmohr  has suggested to remove the arrival process for now to simply the PR. I am removing that
Although, I think that creating VMs using a Poisson process is more realistic, we can include that in the future...

Reply:
@marceloamaral thanks, following up the discussion on the community meeting. I suggested to remove everything which goes beyond just starting X vms. Including all structs where you want to use different functions or values. Just a plain test. Also all watching code and so forth. As @davidvossel, you and I discussed there ,we want to start completely without a framework and just a simple scaling test, starting collecting data in CI.

Reply:
Thanks @rmohr! Right, but I think we agreed to have the thresholds. @rthallisey has written that in the meeting minutes.
To have the thresholds we need to watch the VMIs since we are not including Prometheus. Define the test parameters in a struct is to better organize and visualize the test. IMHO, the PR is not a framework, it is only a test that verifies the VM creation time...

Reply:
@marceloamaral this is probably kind of a misunderstanding, but we argued a lot about separating out all interpretation and thresholds from this PR. I think we all agreed only that we want this in general at some point.

Reply:
@rmohr as we discussed before, I removed all the latency collection, verification and configuration part of the PR.

I only kept the vm watcher to count the number of running VM. As pointed before, Listing operation suffers from API throttling issues, so I kept the watch. But I can also remove that if needed...

Could you please review it again? Let me know if there is still some part of the PR that should be removed...

Reply:
This #6035 kind of tool is what I'm hoping we can begin using for gathering and comparing performance results during these stress tests.